test/integration/targets/filter_urls/runme.yml

```
@@ -0,0 +1,16 @@
+- hosts: testhost
```

Use localhost since no modules are involved (controller execution is all that is needed).

```suggestion
- hosts: localhost
```